libproxy/cmake/modules.cmk

```diff
@@ -18,10 +21,11 @@ include(cmake/modules/network_networkmanager.cmk)
 include(cmake/modules/pacrunner_mozjs.cmk)
 include(cmake/modules/pacrunner_natus.cmk)
 include(cmake/modules/pacrunner_webkit.cmk)
+include(cmake/modules/pacrunner_duktape.cmk)
 
 # Build the pacrunner into libproxy unless we are building for multiple engines
 set(BIPR 1 CACHE BOOL "Build internal pacrunner? (Always false when building more than one PR")
-if((MOZJS_FOUND AND WEBKIT_FOUND) OR (MOZJS_FOUND AND NATUS_FOUND) OR (WEBKIT_FOUND AND NATUS_FOUND))
```

Comment:
Could also do

```
math(EXPR NUM_PACRUNNER "${MOZJS_FOUND}+${NATUS_FOUND}+${WEBKIT_FOUND}+${DUKTAPE_FOUND}")
```

instead of incrementing by one in each file.
```

Reply:
Ha! Of course, let's do that.